### DISCUSSION TITLE
Slack app source code

### DISCUSSION THREAD

**Discussion by amirilovic** [connery-io/connery-sdk#40] [Q&A]
Hi, I’m browsing the code but I couldn’t find anything related to the slack app, is the code for that public?

Thanks!

**Comment by machulav** [comment]
Hi @amirilovic, I appreciate your interest!

This repository contains the source code of the core platform, in particular the runner, which runs plugins and provides a unified API to use them. The Slack app and [the other clients](https://docs.connery.io/docs/native-clients/) are not currently open-sourced while we're exploring options.

However, you can already try using the Slack app hosted by us. Here, you can find more details on how to install and use it: https://docs.connery.io/docs/native-clients/slack

In addition to that, you need to set up the runner from this repository by following our quickstart guide: https://docs.connery.io/docs/quick-start/

If you have any questions or need help, please let me know.
